decidim-admin/config/locales/en.yml

```
@@ -436,6 +436,12 @@ en:
         form:
           save: Save
         success: Help sections updated successfully
+      hidden_resource_mailer:
+        notify_mail:
+          body_1: An administrator removed your proposal because it has been reported as %{reason}
+          body_2: If you feel that has been a mistake please contact the administrators of
+          hello: Hello!
```

```suggestion
          hello: Hello,
```
Looks like this is the way they do it in other mails